src/jira/index.ts

```diff
@@ -41,8 +42,9 @@ const authenticate = async (
 export default (robot: Application): void => {
   const router = robot.route('/jira');
 
+  initializeSentry();
```

Comment:
Do we need to initialize sentry for every route?  Couldn't we just do it the highest level like the start function of main and worker?